src/Navigation/Notifications/Factory/Introduction.php

```diff
@@ -67,7 +67,7 @@ public function __construct(LoggerInterface $logger, Database $dba, BaseURL $bas
 		$this->l10n    = $l10n;
 		$this->pConfig = $pConfig;
 		$this->session = $session;
-		$this->nick    = $app->getLoggedInUserNickname() ?? '';
+		$this->nick    = $session->getLocalUserNickname() ?? '';
```

Comment:
The `App $app` dependency of this class can be removed from the constructor parameter since it's the only place it was used.